src/extension/app/store/app.js

```diff
@@ -961,16 +961,7 @@ export class AppStore {
 
     // update live
     const resp = await this.api.updateLive(path);
-    if (resp) {
-      // bust client cache for live and production
-      if (siteStore.outerHost) {
-        // reuse purgeURL to ensure page relative paths (e.g. when publishing dependencies)
-        await fetch(`https://${siteStore.outerHost}${path}`, { cache: 'reload', mode: 'no-cors' });
-      }
-      if (siteStore.host) {
-        // reuse purgeURL to ensure page relative paths (e.g. when publishing dependencies)
-        await fetch(`https://${siteStore.host}${path}`, { cache: 'reload', mode: 'no-cors' });
-      }
+    if (resp.ok) {
```

Comment:
`updateLive()` can return `null`
```suggestion
    if (resp?.ok) {
```

Reply:
I updated this line to not check for `ok` since it doesn't look like a `Response` is what's returned from `updateLive`.

https://github.com/adobe/aem-sidekick/blob/6c9d4cb1bdc025107156159f848429236b227f57/src/extension/app/utils/admin-client.js#L328

Reply:
No, it's the JSON payload _inside_ the admin response.